src/ansible/roles/packages/tasks/Debian.yml

```diff
@@ -55,6 +55,7 @@
       update_cache: yes
       name:
       - adcli
+      - augeas-tools
```

Comment:
Create a new task called `Install test dependencies on client` after this task and add this dependency there.